src/install/aws/footer.md

```diff
@@ -0,0 +1,5 @@
+</br>
```

Comment:
Do you know if these files accept variables? That way it's truly a template? I can't remember if this is handlebars or what under the covers, but I feel like we should be able to use variables.